### PR TITLE
docker build fix

### PR DESCRIPTION
**Pull request by ZhiHanZ**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary
Fix perf-tool build failure

## Changelog

- Bug Fix

## Related Issues

Fixes #issue

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: .github/workflows/datafuse-perf-docker.yml [hunk 1]
```diff
@@ -13,6 +13,25 @@ jobs:
...
+
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
```
Comment (BohuTANG) [inline .github/workflows/datafuse-perf-docker.yml:28]: This may not a good idea for Rust 2021

Comment (ZhiHanZ) [reply .github/workflows/datafuse-perf-docker.yml:28]: we could bypass all caches by running build on self hosted runner

**Review by BohuTANG — APPROVED** [review]
LGTM

File: .github/workflows/datafuse-performance-dispatch.yaml [hunk 2]
```diff
@@ -51,8 +51,8 @@ jobs:
           cargo build --bin=fuse-query --bin=fuse-store --target x86_64-unknown-linux-gnu
```
Comment (sundy-li) [inline .github/workflows/datafuse-performance-dispatch.yaml:51]: --release

**Pull request merged**